js/models/Settings.js

```diff
@@ -26,10 +30,40 @@ export default class extends BaseModel {
   nested() {
     return {
       shippingAddresses: ShippingAddresses,
-      smtpSettings: SMTPSettings,
```

Comment:
Why was this renamed to 'smtpIntegrationSettings'? It should match what the server calls it which is 'smtpSettings', that way when the model is fetched the right nested instance will be set based off the data returned from the server.